apps/studio/data/lint/lint-query.ts

```diff
@@ -841,13 +844,19 @@ export type ProjectLintsError = unknown
 
 export const useProjectLintsQuery = <TData = ProjectLintsData>(
   { projectRef, connectionString }: ProjectLintsVariables,
```

Comment:
very minor nit: `ProjectLintsVariables` doesn't omit `exposedSchemas` here, so it's possible to pass it in when calling the hook from elsewhere (gets ignored, but could be confusing DX)

Reply:
aah great catch! I updated this — did I do it right @charislam ?

Reply:
yep works now! 👍🏼

Reply:
nice, great catch!